hedera-node/src/main/java/com/hedera/services/ledger/TransferLogic.java

```diff
@@ -226,4 +252,21 @@ private void adjustFungibleTokenAllowance(final BalanceChange change, final Acco
         }
         accountsLedger.set(ownerID, FUNGIBLE_TOKEN_ALLOWANCES, fungibleAllowances);
     }
+
+    /**
+     * Checks if the alias is a known alias i.e, if the alias is already used in any cryptoTransfer
+     * transaction that has led to account creation
+     *
+     * @param change change that contains alias
+     */
+    private void checkIfExistingAlias(final BalanceChange change) {
+        final var alias = change.getNonEmptyAliasIfPresent();
```

Comment:
Major nit 😁, but typically methods named `get*IfPresent()` return `null` as the "missing value" (e.g. `Cache#getIfPresent()` in [Caffeine](https://javadoc.io/doc/com.github.ben-manes.caffeine/caffeine/latest/com.github.benmanes.caffeine/com/github/benmanes/caffeine/cache/Cache.html).

Reply:
Done